src/middlewared/middlewared/api/v25_04_0/api_key.py

```diff
@@ -16,15 +16,16 @@ class AllowListItem(BaseModel):
 
 
 class ApiKeyEntry(BaseModel):
-    """Represents a record in the account.api_key table."""
-
     id: int
     name: Annotated[NonEmptyString, StringConstraints(max_length=200)]
-    key: Private[str]
     created_at: datetime
     allowlist: list[AllowListItem]
 
 
+class ApiKeyEntryWithKey(ApiKeyEntry):
+    key: Private[str]
+
+
 class ApiKeyCreate(ApiKeyEntry):
     id: Excluded = excluded_field()
     key: Excluded = excluded_field()
```

Comment:
Since `ApiKeyEntry` no longer has a `key`, it seems like this line can be removed.